crates/viewer/re_test_viewport/src/lib.rs

```diff
@@ -166,4 +173,21 @@ impl TestContextExt for TestContext {
 
         self.handle_system_commands();
     }
+
+    fn run_view_ui_and_save_snapshot(
+        &mut self,
+        view_id: ViewId,
+        snapshot_name: &str,
+        size: egui::Vec2,
+    ) {
+        let snapshot_options = self.snapshot_options.clone();
```

Comment:
I relate to the desire of having `run_view_ui_and_save_snapshot` to make some of the tests shorter. But having the snapshot options live on the test context live on the test context feels a bit arbitrary to me because:
* size is not on the test context, but the many of the snapshot options are sensitive to it
* this utility is the only thing that uses the options, but its only available on the extensions for views

not a biggy, afaik @lucasmerlin gonna refactor how we set those options soon anyways

Reply:
Good catch. I removed the field, added an argument to the function instead.